lib/appmap/event.rb

```diff
@@ -243,6 +243,7 @@ def build_from_invocation(defined_class, method, receiver, arguments, event: Met
                 value: display_string(value),
                 kind: param_type
                 }.tap do |param|
+                  param[:size] = value.size if value.respond_to?(:size) && value.is_a?(Enumerable)
```

Comment:
Seems like this should be handled by `add_schema`.

Reply:
Yup.